packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -473,7 +473,7 @@ const TagPage = ({classes}: {
                 enableTotal
                 tagId={tag._id}
                 itemsPerPage={200}
-                showNoResults={false}
+                showNoResults={true}
```

Comment:
This came up because the new 2023 Longform Reviews tag had no way to add posts to it, because the PostsList doesn't render until there are posts. Seems wrong for a new tag to not be able to add posts.